### PR TITLE
KAA-993, KAA-1010: Improved Windows support, renamed variables

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @dyosick

### CODE REVIEW

**Comment by iuriigolovach** [comment]
Can one of the admins verify this patch?

File: client/client-multi/client-cpp/tools/build_sdk_thirdparty.bat [hunk 1]
```diff
@@ -35,70 +35,105 @@ goto :eof
 goto :eof
 
 :buildKaaThirdparty
-   echo Building Kaa thirdparty components...
-   call :buildAvro
-   call :buildBotan
+  echo Building Kaa thirdparty components...
+  call :buildZlib
+  call :buildAvro
+  call :buildBotan
```
Comment (ghost) [inline client/client-multi/client-cpp/tools/build_sdk_thirdparty.bat:41]: Need to add sqlite as C++ endpoint uses this database as a log storage.

File: client/client-multi/client-cpp/tools/env.bat [hunk 2]
```diff
@@ -16,14 +16,54 @@
...
+set ZLIB_SRC=zlib-1.2.8
+set ZLIB_HOME=C:\local\%ZLIB_SRC%\%BUILD_PLATFORM%
+set ZLIB_URL=http://zlib.net/zlib-1.2.8.tar.gz
+
+@REM Avro configuration
+set AVRO_SRC=avro-src-1.8.0
+set AVRO_ROOT=C:\local\%AVRO_SRC%\%BUILD_PLATFORM%
+set AVRO_URL=http://archive.apache.org/dist/avro/avro-1.8.0/avro-src-1.8.0.tar.gz
+
+@REM Botan configuration
```
Comment (ghost) [inline client/client-multi/client-cpp/tools/env.bat:53]: Add Sqlite configuration

File: client/client-multi/client-cpp/Modules/FindBotan.cmake [hunk 3]
```diff
@@ -85,11 +85,11 @@ IF( NOT $ENV{BOTAN_LIBRARYDIR} STREQUAL "" )
   SET(_botan_LIBRARIES_SEARCH_DIRS $ENV{BOTAN_LIBRARYDIR} ${_botan_LIBRARIES_SEARCH_DIRS})
 ENDIF( NOT $ENV{BOTAN_LIBRARYDIR} STREQUAL "" )
 
-IF( BOTAN_HOME )
-  SET(_botan_INCLUDE_SEARCH_DIRS ${BOTAN_HOME}/include ${_botan_INCLUDE_SEARCH_DIRS})
-  SET(_botan_LIBRARIES_SEARCH_DIRS ${BOTAN_HOME}/lib ${_botan_LIBRARIES_SEARCH_DIRS})
-  SET(_botan_HOME ${BOTAN_HOME})
-ENDIF( BOTAN_HOME )
+IF( BOTAN_ROOT )
```
Comment (ghost) [inline client/client-multi/client-cpp/Modules/FindBotan.cmake:88]: Edit FindSqlite home folder as well (see FindSqlite3.cmake)

**Comment by RostakaGmfun** [comment]
Moved to #623

**Pull request closed without merging**